core/wiki/macros/translink.tid

```diff
@@ -3,25 +3,13 @@ tags: $:/tags/Macro
 
 \define translink(title,mode:"block")
 \whitespace trim
-<div class={{{ [[tc-translink]] [[$title$]is[missing]then[tc-translink-inline]] [[$mode$]match[block]then[tc-translink-block]] [[$mode$]match[inline]then[tc-translink-inline]] +[first[2]] +[join[ ]] }}}>
+<div class="pre-border" style="padding: 0 1em;">
 <$link to="""$title$""">
-<h1 style="font-weight: bold; font-size: 1.2em;"><$text text="""$title$"""/></h1>
+<h1 class="tc-big-bold"><$text text="""$title$"""/></h1>
 </$link>
 <$transclude tiddler="""$title$""" mode="$mode$">
 "<$text text="""$title$"""/>" is missing
 </$transclude>
+<$list filter="[<__title__>is[missing]] ~[<__mode__>match[inline]]"><p></p></$list>
```

Comment:
Shouldn't be used `:else[<__mode__>match[inline]]` instead of `~[<__mode__>match[inline]]`?

Reply:
It would be clearer, yes, but in the meantime I have found a simpler solution that does not require this list widget at all.